scm-webapp/src/main/java/sonia/scm/api/v2/resources/TagRootResource.java

```diff
@@ -145,6 +203,47 @@ public Response get(@PathParam("namespace") String namespace, @PathParam("name")
     }
   }
 
+  @DELETE
+  @Path("{tagName}")
+  @Produces(VndMediaType.TAG)
+  @Operation(summary = "Delete tag", description = "Deletes the tag provided in the path", tags = "Repository")
+  @ApiResponse(
+    responseCode = "200",
+    description = "success"
+  )
+  @ApiResponse(responseCode = "401", description = "not authenticated / invalid credentials")
+  @ApiResponse(responseCode = "403", description = "not authorized, the current user has no privileges to read the tags")
+  @ApiResponse(
+    responseCode = "404",
+    description = "not found, no tag with the specified name available in the repository",
+    content = @Content(
+      mediaType = VndMediaType.ERROR_TYPE,
+      schema = @Schema(implementation = ErrorDto.class)
+    ))
+  @ApiResponse(
+    responseCode = "500",
+    description = "internal server error",
+    content = @Content(
+      mediaType = VndMediaType.ERROR_TYPE,
+      schema = @Schema(implementation = ErrorDto.class)
+    ))
+  public Response delete(@PathParam("namespace") String namespace, @PathParam("name") String name, @PathParam("tagName") String tagName) {
+    NamespaceAndName namespaceAndName = new NamespaceAndName(namespace, name);
+    try (RepositoryService repositoryService = serviceFactory.create(namespaceAndName)) {
+      RepositoryPermissions.push(repositoryService.getRepository()).check();
+
+      if (tagExists(tagName, repositoryService)) {
+        repositoryService.getTagCommand().delete()
+          .setName(tagName)
+          .execute();
+      }
+
+      return Response.noContent().build();
+    } catch (IOException e) {
+      return Response.serverError().build();
+    }
```

Comment:
`IOExceptions` should be handled by exception mappers elsewhere. It should be safe just to declare them as `throws`.